packages/docs/package.json

```diff
@@ -1,5 +1,11 @@
 {
   "name": "@lest/docs",
   "private": true,
-  "description": "API documentation in JSON form."
+  "description": "API documentation in JSON form.",
+  "main": "dist/src/index.js",
```

Comment:
You have a misconfigured tsconfig if src/index is not emitting to dist/index

Reply:
Ok!